index.html

```diff
@@ -30,12 +22,19 @@
           </li>
         </ul>
         <a href="#" class="mobile-nav">
-          <img
-            class="navImage"
-            src="./Images/ic_menu.svg"
-            alt="Navigation menu"
-          />
+          <img class="navImage" src="./Images/ic_menu.svg" alt="Navigation menu" />
         </a>
+        <div class="header__menu">
+          <button class="header__menu-btn">
+            <img src="images/Icon-Cancel.png" alt="Menu cancel button" />
+          </button>
+
+          <ul class="header__menu-content">
+            <li><a class="header__menu-link" href="#portfolio">Portfolio</a></li>
+            <li><a class="header__menu-link" href="#about">About</a></li>
+            <li><a class="header__menu-link" href="#contact">Contact</a></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Kindly make sure that the user cannot scroll in the page itself when the mobile menu is open. Currently, if you have the mobile menu open, you can scroll past it onto the page content.